docs/docs/start/docker_install.md

```diff
@@ -247,7 +247,7 @@ index 8adee63..dc3993c 100644
 -        image: inventree/inventree:${INVENTREE_TAG:-stable}
 +        image: inventree/inventree:${INVENTREE_TAG:-stable}-custom
```

Comment:
Q: Is this intended to be included?

Reply:
No, nice catch..

Reply:
I think that the  github diff preview is being tricked, as we have a "diff" in the documentation...